packages/sdks/javascript/README.md

```diff
@@ -106,19 +106,25 @@ make test
 ```
 
 ### Without Docker
+Installing on ``Linux`` or ``macOS`` systems.
 
-#### 1. Install missing dependencies
+#### 1. Navigate to the project directory!
+```
+.\packages\sdks\javascript\
+```
```

Comment:
Please use forward slash and not backslash. We officially support `Unix` based operating systems (such as Linux and macOS) so all documentation should give examples that work on those systems.
If you're using other operating systems, e.g. `Windows`, it's recommended that you use [WSL](https://ubuntu.com/wsl) and run Linux commands from Linux.

```suggestion
```shell
cd packages/sdks/javascript/
```

That been said, `Docker` is the preferred and recommended approach and allows building this SDK in any operating system compatible with Docker.